packages/@aws-cdk/aws-lambda/lib/permission.ts

```diff
@@ -64,4 +69,11 @@ export interface Permission {
    * can invoke the function.
    */
   readonly sourceArn?: string;
+
+  /**
+   * The authType for the function URL that you are granting permissions for.
+   *
+   * @default - No functionUrlAuthType
+   */
+  readonly functionUrlAuthType?: FunctionUrlAuthType;
```

Comment:
Seems like the only use case for this parameter is to distinguish between:

- `*` (NONE)
- `*` (AWS_IAM)

No? 

So, difference between anyone anonymously, or anyone authenticated? No need to pass it otherwise?